test/expect/dune

```diff
@@ -6,6 +6,7 @@
  (modes byte)
  (libraries
   unix
+  dynlink
```

Comment:
Do you remember why you had to add `dynlink` here? Which version of `dune` are you using?